moja-kuhinjica-frontend/src/components/label/ErrorLabel.tsx

```diff
@@ -1,12 +1,37 @@
-import React from 'react'
+import React, { useEffect, useState } from 'react'
+import { useTranslation } from 'react-i18next'
 
 interface IErrorLabelProps {
     content: string | undefined
     style?: string
 }
+
+const COMAPRING_NUMBER = 4
+
 export const ErrorLabel = ({
     content,
     style,
 }: IErrorLabelProps): JSX.Element => {
-    return <label className={`errorLabel ${style}`}>{content}</label>
+    const { t } = useTranslation()
+    const [beforeEmail, setBeforeEmail] = useState<string>('')
+    const [email, setEmail] = useState<string>('')
+    const [afterEmail, setAfterEmail] = useState<string>('')
```

Comment:
this can be single state
for example
```
const [emailMatchValues, setEmailMatchValues] = useState<{
  beforeEmail: string
  email: string
  afterEmail: string
}>({
  beforeEmail: '',
  email: '',
  afterEmail: ''
})
```